src/app/public/modules/demo/demo.component.spec.ts

```diff
@@ -202,7 +202,7 @@ describe('Demo component', () => {
         );
       }
 
-      const mockAuthProvider = TestBed.get(SkyAuthTokenProvider) as DemoAuthTokenMockProvider;
+      const mockAuthProvider = TestBed.inject(SkyAuthTokenProvider) as unknown as DemoAuthTokenMockProvider;
```

Comment:
You can keep the type checking by removing the `as unknown` and putting an optional indicator on the `testToken` property in `DemoAuthTokenMockProvider`.
```
public testToken?: SkyAuthToken
```

Reply:
Ah got it. Thanks for the tip. Done.